katago_server/games/serializers/rating_game.py

```diff
@@ -2,27 +2,27 @@
 
 from rest_framework.serializers import HyperlinkedModelSerializer, HiddenField, CurrentUserDefault
 
-from katago_server.games.models import RankingEstimationGame
+from katago_server.games.models import RatingGame
 from katago_server.trainings.serializers import NetworkSerializerForTasks
 from katago_server.users.serializers import LimitedUserSerializer
 
 
 # Use as write only serializer
-class RankingEstimationGameCreateSerializer(HyperlinkedModelSerializer):
+class RatingGameCreateSerializer(HyperlinkedModelSerializer):
     # https://www.django-rest-framework.org/api-guide/validators/#advanced-field-defaults
     submitted_by = HiddenField(default=CurrentUserDefault())
 
     class Meta:
-        model = RankingEstimationGame
+        model = RatingGame
         fields = "__all__"
 
 
 # Use as read only serializer
-class RankingEstimationGameListSerializer(HyperlinkedModelSerializer):
+class RatingGameListSerializer(HyperlinkedModelSerializer):
     submitted_by = LimitedUserSerializer()
     white_network = NetworkSerializerForTasks()
     black_network = NetworkSerializerForTasks()
 
     class Meta:
-        model = RankingEstimationGame
+        model = RatingGame
```

Comment:
While we are here we should:

1)    add a docstring to the serializer
2)   be explicit in the fields
3)    and include 'url' field as mentioned above

Reply:
Thanks.